portal/portal-service-impl/impl/src/java/org/sakaiproject/portal/service/PortalServiceImpl.java

```diff
@@ -618,5 +647,181 @@ public String getSkinPrefix() {
 		return "";
 	}
 
+	public List<BullhornAlert> getSocialAlerts(String userId) {
+
+		List<BullhornAlert> alerts = sqlService.dbRead(
+                "SELECT * FROM SOCIAL_ALERTS WHERE TO_USER = ? ORDER BY EVENT_DATE DESC"
```

Comment:
I agree, if it is used in two places. It isn't though. Until you need to worry about duplication and divergence I just think it's a lot more readable to just have it there with the code that uses it.